pom.xml

```diff
@@ -187,7 +187,7 @@
                 <configuration>
                     <cloneProjectsTo>${project.build.directory}/it</cloneProjectsTo>
                     <localRepositoryPath>${project.build.directory}/local-repo</localRepositoryPath>
-                    <settingsFile>src/it/settings.xml</settingsFile>
+                    <!-- settingsFile>src/it/settings.xml</settingsFile -->
                     <goals>
```

Comment:
But probably hard to implement. I think you detect variables withing string - and have to encode them for saving the resulting file. 

Another approach would be a configurable method you call before closing the tar file. I'd add my binary file in this method. And make my method availabe to your Code by adding a dependency to the Plugin config in my pom.